api/PaymentAddress.json

```diff
@@ -551,56 +551,6 @@
           }
         }
       },
-      "regionCode": {
-        "__compat": {
-          "mdn_url": "https://developer.mozilla.org/docs/Web/API/PaymentAddress/regionCode",
-          "support": {
-            "chrome": {
-              "version_added": false
-            },
-            "chrome_android": {
-              "version_added": false
-            },
-            "edge": {
-              "version_added": false
-            },
-            "firefox": {
-              "version_added": false,
-              "notes": "Available only in nightly builds."
-            },
-            "firefox_android": {
-              "version_added": false,
-              "notes": "Available only in nightly builds."
```

Comment:
There are several notes here like this, but I guess we can remove them since the interface itself is deprecated and thus we don't expected them to be shipped.

Reply:
@Elchi3 do you have a link to the recent issue about representing preview browsers? I can't find it...

But in any case I agree with removing this one.

Reply:
Removing sounds good to me, too. 
The proposal on "preview" is here: https://github.com/mdn/browser-compat-data/pull/10334 (and I think it would address this use case)